Update dallas_temp.md (#5847)
OttoWinter/esphomedocs: content/components/sensor/dallas_temp.md

- All other options from [Sensor](/components/sensor).

> ### Byte Order of the Address 

Reviewer: pls fix

> EspHome (most recent version 2026-January) - uses reversed order of the bytes, compared to the Arduino IDE. That means, if you have an address for a sensor via the Arduino IDE that says 0x2870F5E0161301C7 - then you have to use 0xc7011316e0f57028 in EspHome. The bytes are pairwise ordered in reverse. The Arduino IDE version ends in the above example with c7 - the EspHome version the same address starts with c7. - The other pairs accordingly. The arduino IDE is LSB‑first. While EspHome is MSB‑first.

Reviewer: ```suggestion
ESPHome starting with 2026.1.0 uses reversed order of the bytes, compared to the Arduino IDE. That means, if you have an address for a sensor via the Arduino IDE that says `0x2870f5e0161301c7`, then use `0xc7011316e0f57028` in ESPHome. The bytes are pairwise ordered in reverse. The Arduino IDE version ends in the above example with `c7`, the ESPHome version the same address starts with `c7`. The other pairs accordingly. The Arduino IDE is LSB‑first, while ESPHome is MSB‑first.
```

Reviewer 2: Why is there an esphome version mentioned at all?  It has always done it this way.

> If you don't have the addresses, use index instead of addresses for the first run. EspHome will print the addresses on the log at first use with index. After that, try with addresses, if you want.

Reviewer: ```suggestion
If you don't have the addresses, use index instead of addresses for the first run. ESPHome will print the addresses on the log at first use with index. If you have the addresses its recommended to use a configuration with them.
```

Reviewer 2: If you don't have the addresses, then just run it without any sensors configured.  It will always print the detected addresses in the log.
